src/traces/pointcloud/attributes.js

```diff
@@ -0,0 +1,124 @@
+/**
+* Copyright 2012-2016, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+'use strict';
+
+var scatterglAttrs = require('../scattergl/attributes');
+
+module.exports = {
+    x: scatterglAttrs.x,
+    y: scatterglAttrs.y,
+    xy: {
+        valType: 'data_array',
+        description: [
+            'Faster alternative to specifying `x` and `y` separately.',
+            'If supplied, it must be a typed `Float32Array` array that',
+            'represents points such that `xy[i * 2] = x[i]` and `xy[i * 2 + 1] = y[i]`'
+        ].join(' ')
+    },
+    indexid: {
```

Comment:
@monfera do the `indexid` items have to be numbers? 

If so, I'd vote for naming this attribute `indices`. If not, I'd